src/Apps/Consign/Hooks/useSubmissionFlowSteps.ts

```diff
@@ -0,0 +1,35 @@
+import { themeProps } from "@artsy/palette"
+import { useFeatureFlag } from "System/useFeatureFlag"
+import { __internal__useMatchMedia } from "Utils/Hooks/useMatchMedia"
+
+function typedArray<T extends string>(...elems: T[]): T[] {
+  return elems
+}
```

Comment:
What's this all about? It looks like you just want `["Foo", "Bar"] as const`?

Reply:
Yes, it is the same 👍 . Do you think it is cleaner to just do `["Foo", "Bar"] as const`? With that approach I have to add `as const` to every array I return in  useSubmissionFlowSteps in order to correctly type the `currentStep` prop in `SubmissionStepper`

Reply:
Yep that's definitely preferred than adding code like this. That's a feature of TypeScript — we should use it.

Reply:
Though I definitely feel like I'm missing something here as to why that would be necessary. Either way: I would strongly endeavor to remove something like this.

Reply:
It is not exactly necessary. It just makes typing the `currentStep` prop easier and reads better in VSCode because hovering it tells exactly which String Literal is expected as opposed to showing the type name.
But I will remove it.